src/components/Nav/NavItemLink/NavItemLink.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file Nav item link.
+ * @copyright IBM Security 2019
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const NavItemLink = React.forwardRef(function NavItemLink(props, ref) {
+  const { element, ...rest } = props;
+  return React.createElement(element, { ...rest, ref });
+});
+
+NavItemLink.displayName = 'Link';
```

Comment:
Is this line renaming the display name from `NavItemLink` to `Link`? If so, I think it's good to leave it as-is to avoid any confusion in test output - https://github.com/carbon-design-system/ibm-security/pull/155/files#diff-061f99714b39a9b07ec0dd9e79fd3ae7R12